build/README.md

```diff
@@ -404,9 +404,9 @@ a PR to that effect would be greatly appreciated.
 1.  If joining an existing pool, share your CRDB node addresses with the
     operators of the existing DSS instances.  They will add these node addresses
     to JoinExisting where `VAR_CRDB_EXTERNAL_NODEn` is indicated in the minimum
-    example, and then perform another rolling restart of their CockroachDB pods:
+    example, and then update their deployment:
 
-    `kubectl rollout restart statefulset/cockroachdb --namespace $NAMESPACE`
+    `tk apply workspace/$CLUSTER_CONTEXT`
```

Comment:
I would recommend to keep the rollout restart as an hint in case the tk apply was done at the wrong time.

Reply:
There is a rolling restart command identical to this in step "If joining an existing DSS pool" (third substep), but it's not appropriate here -- the instructions in this step are to change JoinExisting, which is a value in the Tanka main.jsonnet.  That change won't get adopted by performing a rolling restart because, until `tk apply`, that change is merely a local change.

Reply:
Make sense 👍